aat/sessionmeta_test.go

```diff
@@ -264,13 +262,17 @@ func TestMetaProcSessionCount(t *testing.T) {
 		t.Fatal("Session count should be same as first")
 	}
 
-	err = caller.Close()
+	subscriber.Unsubscribe(metaOnJoin)
+	subscriber.Unsubscribe(metaOnLeave)
```

Comment:
Not a requirement at all.  I guess there is no need, especially since I am not checking for error on unsubscribe.